Dockerfile/Dockerfile

```diff
@@ -0,0 +1,71 @@
+# Ubuntu Trusty Docker file for base TileDB Dependencies
+
+# This Dockerfile copies files from the parent directory, requiring a build
+# context from the parent directory. From `./..`, execute the following to
+# build:
+# docker build -f Dockerfile/Dockerfile -t tiledb:dev .
+#
+# Use the build arg 'enable' to configure optional TileDB components, e.g.:
+#   docker build -f Dockerfile/Dockerfile --build-arg enable=s3,hdfs -t tiledb:dev
+
+# Ubuntu Trusty
```

Comment:
18.04 is actually Bionic (IIRC Xenial was 16.04, Trusty 12.034). The current LTS is 20.04 aka Focal which works fine.

Reply:
20.04 works, updated